lib/src/analysis_options.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+const String analysisOptions = '''
+analyzer:
+  strong-mode: true
+
+# Source of linter options:
+# http://dart-lang.github.io/linter/lints/options/options.html
+
+linter:
+  rules:
+    - annotate_overrides
+    - hash_and_equals
+    - valid_regexps
```

Comment:
Annotate overrides is still not something we do consistently in our own code. I'd just keep these two...

Reply:
ack